git_machete/docs.py

```diff
@@ -341,26 +341,25 @@
           <b>Options:</b>
             <b>--draft</b>    Creates the new PR as a draft.
 
-        <b>`retarget-pr`:</b>
-
-          Sets the base of the current branch's PR to upstream (parent) branch, as seen by git machete (see `git machete show up`).
-
-        <b>`fetch-prs <PR-number-1> ... <PR-number-N> [--all] [--mine] [--by]`:</b>
+        <b>`fetch-prs[ --all | --by=<github-login> | --mine | <PR-number-1> ... <PR-number-N>`:</b>
 
           Check out the head branch of the given pull requests (specified by number),
           also traverse chain of pull requests upwards, adding branches one by one to git-machete and check them out locally.
           Once pull request is checked out locally, annotate local branches with corresponding pull request numbers.
           (Plural version of the `checkout-prs` subcommand but without switching branches at the end)
 
-          <b>Parameters:</b>
-            <b><PR-number-1> ... <PR-number-N></b>    Pull request numbers to checkout.
-
           <b>Options:</b>
             <b>--all</b>     Checkout all open PRs.
-            <b>--mine</b>    Checkout mine open PRs.
             <b>--by</b>      Checkout someone's open PRs.
               <b>Parameters:</b>
-                <b> <Github login></b> Github account name.
+                <b>github-login></b> Github account name.
+            <b>--mine</b>    Checkout mine open PRs.
```

Comment:
Whoops `mine` can't be used in such a position... rather `my`. But I'd make it even more precise:
```suggestion
            <b>--mine</b>    Checkout open PRs for the current user associated with the Github token.
```
coz that's what "my"/"I" really means here.